api.ts

```diff
@@ -0,0 +1,53 @@
+import { inDevMode } from "./utils";
+
+export class DmpApi {
+  getUrl() {
+    let apiHost = inDevMode() ? 'api.dmphub.uc3dev.cdlib.net' : `api.${window.location.hostname}`;
```

Comment:
This is good for now, but we should switch this to use GraphQL in the future. My intention is to let the old DMPTool use those API endpoints and for the new system to use GraphQL. So we will transition away from supporting both.